src/components/themed/CreateWalletSelectCryptoRow.tsx

```diff
@@ -38,7 +38,9 @@ export const CreateWalletSelectCryptoRowComponent = (props: Props) => {
 
   const account = useSelector(state => state.core.account)
   const currencyConfigs = useWatch(account, 'currencyConfig')
-  const { currencyCode } = tokenId != null ? currencyConfigs[pluginId].allTokens[tokenId] : currencyConfigs[pluginId].currencyInfo
+  const isToken = tokenId != null
+  const { currencyCode } = isToken ? currencyConfigs[pluginId].allTokens[tokenId] : currencyConfigs[pluginId].currencyInfo
```

Comment:
Do not destruction currencyCode immediately. Check if `currencyConfigs[pluginId].allTokens[tokenId]` is non-null or you might crash

Reply:
Breaks the hook usage below. Will add another check